source/pda_aq_programmer.c

```diff
@@ -754,7 +754,11 @@ void *set_aqualink_PDA_init( void *ptr )
     //printf("****** Version '%s' ********\n",aq_data->version);
     LOG(PDA_LOG,LOG_DEBUG, "PDA type=%d, version=%s\n", _PDA_Type, aq_data->version);
     // don't wait for version menu to time out press back to get to home menu faster
-    //send_pda_cmd(KEY_PDA_BACK);
+    #if AQ_PDA
+      if (_aqconfig_.pda_bypass_info) {
```

Comment:
Honestly I think this should be the default behavior for PDA and not optional.  I believe the root cause of the issue is the info screen does not appear if filter pump is not running.  It's faster and more reliable to skip waiting on info and go straight to loop over devices.  The additional information on the info screen is for SWG which you can get by reading the SWG messages directly.

Reply:
also need to remove comments around loopover_devices